pkg/webhook/resources/setting/validator_test.go

```diff
@@ -1010,6 +1010,22 @@ func Test_validateUpgradeConfig(t *testing.T) {
 			},
 			expectedErr: false,
 		},
+		{
+			name: "LogeReadyTimeout not in expected range",
```

Comment:
name: ... is in expected range

Reply:
Weird, I thought I already did https://github.com/harvester/harvester/pull/8421#discussion_r2190304745.